packer/HWIMO-BUILD

```diff
@@ -119,6 +120,19 @@ else
    fi
 fi
 
+### Choose a deb repository/component , according to IS_OFFICIAL_RELEASE parameter
+if [ $ANSIBLE_PLAYBOOK != "rackhd_local" ]; then
+       if [ ! -n "${DEBIAN_REPOSITORY}" ];  then
+           DEBIAN_REPOSITORY="deb https://dl.bintray.com/rackhd/debian trusty release"
```

Comment:
There's default value in playbook so why set again here?

Reply:
fixed